demo.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+from enum import IntEnum
+
+from telegram import ReplyKeyboardMarkup, ReplyKeyboardRemove
+from telegram.ext import CommandHandler, MessageHandler, Filters, ConversationHandler
+
+
+class DemoDecisions(IntEnum):
+    FAVPIC = 1
+    FAMILY = 2
+    AMAZON = 3
+    DURABLE = 4
+    DIFFICULT = 5
+    MICHAEL = 6
+    WINNER = 7
```

Comment:
Imo, this looks nicer (and takes less lines). Why are we starting with 1?

```suggestion
    FAVPIC, FAMILY, AMAZON, DURABLE, DIFFICULT, MICHAEL, WINNER = range(1, 8)
```

Reply:
tbh I don't like that. When adding new values, we need to add the value and change the range function's parameters. I get the idea, but imho it's better maintainable to just use multiple lines.

Reply:
I'm @d-Rickyy-b on this, keep it simple. I would start with 0 though, otherwise we'll be working with mathematicians only soon.